CHANGELOG.md

```diff
@@ -40,8 +42,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Fixed label selector for webhook and manager services.
 
-[Unreleased]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.5.3...HEAD
-[0.5.3]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.5.2...v0.5.3
+[Unreleased]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.5.2...HEAD
 [0.5.2]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.5.2...v0.5.2
```

Comment:
this line needs to be removed as well

Reply:
The line I mean is comparing with itself. This needs to be removed
```
[0.5.2]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.5.2...v0.5.2
```

because we already have this right below

```
[0.5.2]: https://github.com/giantswarm/cluster-api-provider-azure-app/compare/v0.4.12-gsalpha3...v0.5.2
```

Reply:
of course, good catch